hat/app/org/hatdex/hat/she/mappers/InstagramMappers.scala

```diff
@@ -12,18 +12,30 @@ import scala.util.Try
 
 class InstagramMediaMapper extends DataEndpointMapper {
   override protected val dataDeduplicationField: Option[String] = Some("id")
-
   def dataQueries(fromDate: Option[DateTime], untilDate: Option[DateTime]): Seq[PropertyQuery] = {
     val unixDateFilter = fromDate.flatMap { _ =>
       Some(FilterOperator.Between(Json.toJson(fromDate.map(t => (t.getMillis / 1000).toString)), Json.toJson(untilDate.map(t => (t.getMillis / 1000).toString))))
     }
 
-    Seq(PropertyQuery(
+    val propertyQueryv1 = PropertyQuery(
+      List(EndpointQuery("instagram/feed", None,
+        unixDateFilter.map(f ⇒ Seq(EndpointQueryFilter("created_time", None, f))), None)), Some("created_time"), Some("descending"), None)
+
+    val propertyQueryv2 = PropertyQuery(
       List(EndpointQuery("instagram/feed", None,
-        unixDateFilter.map(f ⇒ Seq(EndpointQueryFilter("created_time", None, f))), None)), Some("created_time"), Some("descending"), None))
+        unixDateFilter.map(f ⇒ Seq(EndpointQueryFilter("ds_created_time", None, f))), None)), Some("ds_created_time"), Some("descending"), None)
```

Comment:
I know that it's a big chunk of work up front, but using `stringly-typed` matchers is not ideal.
I'd suggest defining `final val createdDate = "ds_created_time"`

Reply:
It's nicer for maintenance as well. :)

Reply:
Same comment for the other situations here as well.

Reply:
I agree. It was part of my suggestion to Product for the feed mappers. The mapper needs 2 fields. A date field to sort things and an unique field that can be used to deduplicate fields. We can either have it as a variable or even as a Map[String: String] for all the plugs and apps that we have in the feed. That didn't move though and it's now blocked since product focuses in getting the Facebook plug to pass review

Reply:
I think we can approve this and create an issue to convert strings to variables or maps in the future :)

Reply:
please create an issue for this fix